book/src/example.md

```diff
@@ -3,136 +3,26 @@
 > ## *This page is out-of-date and under active development.*
 
 The
-[HostPath](https://github.com/kubernetes-csi/drivers/tree/master/pkg/hostpath)
-can be used to provision local storage in a single node test. This
-section shows how to deploy and use that driver in Kubernetes.
+[HostPath](https://github.com/kubernetes-csi/csi-driver-host-path)
+can be used to provision local storage. The deployment example there
```

Comment:
"deployment example there": can you link to the actual page?

Reply:
I will make "deployment example there" to be a link to actual deployment script, if that's what you mean